microservice/src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+server:
+  port: 0
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost/skytask
+    username: postgres
+    password: postgres
+  jpa:
+    generate-ddl: true
+
+  rabbitmq:
+    host: localhost
```

Comment:
Put local host has a property and then use it in rabbit.host and spring.dataSource.url, find a way to re use yml properties inside others.